plugins/org.eclipse.sirius.table/src/org/eclipse/sirius/table/business/internal/refresh/DTableSynchronizerImpl.java

```diff
@@ -17,25 +17,24 @@
 import java.util.HashMap;
```

Comment:
Same remarks than for DTableElementSynchronizer

Reply:
Code coverage gives the same level of coverage before and after the refactoring.
Following segments of case were/are not covered:

In DTableElementSynchronizer
- column with default width == 0
- line without target
- refreshTarget for Cell of CrossTable

Reply:
Same remark (to be done later)